recipes/openexr/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class OpenEXRConan(ConanFile):
+    name = "openexr"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/openexr/openexr"
+    topics = ("conan")
+    license = "BSD-3"
+    description = "OpenEXR is a high dynamic-range (HDR) image file format for use in computer imaging applications"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False], "namespace_versioning": [True, False]}
+    default_options = {"shared": False, "namespace_versioning": True, 'fPIC': True}
+    generators = "cmake"
+
+    exports_sources = ["CMakeLists.txt"]
+
+    _source_subfolder = "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "openexr-{}".format(self.version)
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+
+    def requirements(self):
+        self.requires.add("zlib/1.2.11")
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["ZLIB_LIBRARY"] = self.deps_cpp_info["zlib"].libs[0]
+        cmake.definitions["ZLIB_INCLUDE_DIR"] = self.deps_cpp_info["zlib"].include_paths[0]
+        cmake.definitions["OPENEXR_BUILD_PYTHON_LIBS"] = False
+        cmake.definitions["BUILD_TESTING"] = False
+        cmake.definitions["OPENEXR_VIEWERS_ENABLE"] = False
+        cmake.definitions["OPENEXR_BUILD_SHARED"] = self.options.shared
+        cmake.definitions["OPENEXR_BUILD_STATIC"] = not bool(self.options.shared)
```

Comment:
Explicit conversion not needed
```suggestion
        cmake.definitions["OPENEXR_BUILD_STATIC"] = not self.options.shared
```